src/xargs/mod.rs

```diff
@@ -813,11 +841,21 @@ fn do_xargs(args: &[&str]) -> Result<CommandResult, XargsError> {
         )
         .arg(
             Arg::new(options::VERBOSE)
-                .short('t')
-                .long(options::VERBOSE)
-                .help("Be verbose")
-                .action(ArgAction::SetTrue),
+            .short('t')
+            .long(options::VERBOSE)
+            .help("Be verbose")
+            .action(ArgAction::SetTrue),
         )
+        .arg(
+            Arg::new(options::REPLACE)
+                .long(options::REPLACE)
+                .short('I')
+                .short_alias('i')
+                .num_args(0..=1)
+                .value_parser(clap::value_parser!(String))
+                .help("Replace R in INITIAL-ARGS with names read from standard input; if R is unspecified, assume {}"),
```

Comment:
This line breaks formatting with `fmt`.
```suggestion
                .help(
                    "Replace R in INITIAL-ARGS with names read from standard input; \
                    if R is unspecified, assume {}",
                ),
```